### PR TITLE
Add database migrations

### PR DESCRIPTION
**Pull request by vsujan**
- Add knex dependency
- Add migration and seed scripts.
- Add postgres package.
- Add environment variables.
- Add knex file.
- Add empty folder for migrations and seeds.

@kabirbaidhya

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -11,14 +11,21 @@
     "clean": "rimraf dist/",
     "build": "run-s lint clean babel",
     "babel": "babel src/ -d dist/",
-    "list:routes": "babel-node ./resources/scripts/list-routes.js"
+    "list:routes": "babel-node ./resources/scripts/list-routes.js",
+    "migrate:make": "knex migrate:make",
+    "migrate:latest": "knex migrate:latest",
```
Comment (kabirbaidhya) [inline package.json]: Let's make this script just `migrate`. So, that we can run it with just `npm run migrate`.

File: package.json [hunk 2]
```diff
@@ -11,14 +11,21 @@
...
+    "migrate:rollback": "knex migrate:rollback",
+    "seed:make": "knex seed:make",
+    "seed:run": "knex seed:run"
```
Comment (kabirbaidhya) [inline package.json]: And let's make it `seed` so that we can run it with just `npm run seed`.

File: package.json (hunk 1 above)
Comment (vsujan) [reply package.json]: 🆗

File: package.json (hunk 2 above)
Comment (vsujan) [reply package.json]: 🆗

**Review by kabirbaidhya — APPROVED** [review]

**Pull request merged**